src/pal/tests/palsuite/c_runtime/__iscsym/test1/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ add_executable(paltest_iscsym_test1
 add_dependencies(paltest_iscsym_test1 coreclrpal)
 
 target_link_libraries(paltest_iscsym_test1
-  pthread
```

Comment:
I don't think we can remove pthread unconditionally. On Linux, if it is not linked with, some synchronization primitives like mutexes don't work, they just become dummy.